src/orion/core/evc/experiment.py

```diff
@@ -86,11 +87,12 @@ def parent(self):
         if self._parent is None and self._no_parent_lookup:
             self._no_parent_lookup = False
             query = {'_id': self.item.refers['parent_id']}
-            selection = {'name': 1}
+            selection = {'name': 1, 'version': 1}
             experiments = Database().read('experiments', query, selection=selection)
             if experiments:
-                self.set_parent(ExperimentNode(name=experiments[0]['name']))
-
+                parent = experiments[0]
+                exp_node = ExperimentNode(name=parent['name'], version=parent['version'])
+                self.set_parent(exp_node)
```

Comment:
Any idea why this problem never showed up before? We should add a test for this in test_experiment_tree.